priv/app/call_tracer.jsx

```diff
@@ -118,12 +118,8 @@ export default class CallsTracer extends React.Component {
     };
   }
 
-  componentDidMount() {
```

Comment:
this is an edge case but if someone starts capturing long calls in one window for the first time for that MFA then it wont be visible in the second window (becase CallTracer in the 2nd window does not do polling, if you remove this)
I understand this does unnecessary polling in 99% of cases, but I would keep the old behaviour for now, and improve this in the optimisation/rewrite PR